migrations/versions/9dee95275ff0_.py

```diff
@@ -1,153 +0,0 @@
-"""empty message
```

Comment:
Shoud we delete this migration?

Reply:
We should delete this migration, I also deleted it in my PR, would probably create conflitcs